### DISCUSSION TITLE
What defines a RMM

### DISCUSSION THREAD

**Discussion by WesSec** [magicsword-io/LOLRMM#37] [General]
Can we maybe define what a RMM is, what are its prerequisites? 

Reason for this discussion: Rapid7 (EDR solution) is marked as RMM in the project, while i believe its not, but i want to reach consensus before we start throwing back and forth pull requests.

**Comment by nasbench** [comment]
That Rapid 7 tool along with others were added by mistake and will be removed. An RMM is what the industry defines as an RMM, i.e having probably agents, the ability to launch remote commands through them and all of those common traits.

If you want to submit a PR and you don't know if the tool fits, you can always open an issue and ask.

Cheers.
